### DISCUSSION TITLE
MacOS update caused bookmarks disappearance

### DISCUSSION THREAD

**Discussion by phonoarchive** [iterate-ch/cyberduck#16350] [Q&A]
Hello,

I've just updated my MacOS to Sequoia and now I can't find my bookmarks in Cyberduck. Please help! :)

**Comment by dkocher** [comment]
The bookmarks should still be loaded from the _Bookmarks_ folder in the [application support directory](https://docs.cyberduck.io/cyberduck/support/#application-support-folder).

**Reply by phonoarchive** [reply]
> The bookmarks should still be loaded from the _Bookmarks_ folder in the [application support directory](https://docs.cyberduck.io/cyberduck/support/#application-support-folder).

`~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Bookmarks` shows no files inside _Bookmarks_ folder... I checked all folders inside _duck_ folder, all of them are empty.

**Reply by dkocher** [reply]
I have no explanation for this. Please see if you can recover the configuration from backup.

**Reply by taricco** [reply]
I can confirm I have the same issue with Cyberduck after upgrading to MacOS Sequoia. My bookmarks are gone.

**Reply by tatze96** [reply]
It is expected that Cyberduck shows the same behavior in this case, as Mountain Duck and Cyberduck draw the bookmarks from the identical folder.
